src/bp/ui-studio/src/web/views/FlowBuilder/diagram/index.tsx

```diff
@@ -183,7 +219,7 @@ export default class FlowBuilder extends Component<Props> {
       }
     }
 
-    this.props.openFlowNodeProps()
+    toastInfo('Pssst! Just click once a node to inspect it, no need to double-click anymore.', Timeout.LONG)
```

Comment:
I'm not absolutely sure this is necessary, since they will definitely see it on the screen. We added a message for ctrl+s since the auto-save was silent and it was not clear that it was auto-saving.

Reply:
Let's keep it there for now, and remove it in 12.2.1 . I added a todo: https://github.com/botpress/botpress/pull/2482/commits/927fa5cf131d681710d539821c0dffae4664b759